src/examples/files.jl

```diff
@@ -13,7 +13,7 @@ function validpath(root, path; dirs = true)
 end
 
 ormatch(r::RegexMatch, x) = r.match
-ormatch(r::Nothing, x) = x
+ormatch(r::Void, x) = x
```

Comment:
`Void` is `None` on Julia 0.3, so this method will not match any arguments.

One way to fix this is to make a new no-thing thing `const Nonce = VERSION < v"0.4-dev" ? Nothing : Void`